### PR TITLE
qa/tests: add light octopus-x (cephadm based)

### PR DESCRIPTION
**Pull request by yuriw**
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by yuriw** [comment]
`teuthology-suite -v --ceph-repo https://github.com/ceph/ceph.git --suite-repo https://github.com/yuriw/ceph.git -c master -m $MACHINE_NAME -s upgrade/octopus-x -k $DISTRO -p $PRIO -e $CEPH_QA_MAIL --suite-branch wip-yuriw-octopus-x-3-master -l $LIMIT --force-priority -n 7`

http://pulpito.front.sepia.ceph.com:80/yuriw-2020-12-18_17:04:54-upgrade:octopus-x-master-distro-basic-smithi/

**Comment by neha-ojha** [comment]
@yuriw 

https://pulpito.ceph.com/yuriw-2020-12-18_17:04:54-upgrade:octopus-x-master-distro-basic-smithi/5718746/ failed with https://tracker.ceph.com/issues/48142

https://pulpito.ceph.com/yuriw-2020-12-18_17:04:54-upgrade:octopus-x-master-distro-basic-smithi/5718747/ the upgrade did not succeed

```
2020-12-18T17:24:58.316 INFO:teuthology.orchestra.run.smithi107.stdout:{   
2020-12-18T17:24:58.317 INFO:teuthology.orchestra.run.smithi107.stdout:    "mon": {
2020-12-18T17:24:58.317 INFO:teuthology.orchestra.run.smithi107.stdout:        "ceph version 15.2.0 (dc6a0b5c3cbf6a5e1d6d4f20b5ad466d76b96247) octopus (rc)": 3
2020-12-18T17:24:58.317 INFO:teuthology.orchestra.run.smithi107.stdout:    },
2020-12-18T17:24:58.317 INFO:teuthology.orchestra.run.smithi107.stdout:    "mgr": {
2020-12-18T17:24:58.318 INFO:teuthology.orchestra.run.smithi107.stdout:        "ceph version 15.2.0 (dc6a0b5c3cbf6a5e1d6d4f20b5ad466d76b96247) octopus (rc)": 2
2020-12-18T17:24:58.318 INFO:teuthology.orchestra.run.smithi107.stdout:    },
2020-12-18T17:24:58.318 INFO:teuthology.orchestra.run.smithi107.stdout:    "osd": {
2020-12-18T17:24:58.318 INFO:teuthology.orchestra.run.smithi107.stdout:        "ceph version 15.2.0 (dc6a0b5c3cbf6a5e1d6d4f20b5ad466d76b96247) octopus (rc)": 8
2020-12-18T17:24:58.319 INFO:teuthology.orchestra.run.smithi107.stdout:    },
2020-12-18T17:24:58.319 INFO:teuthology.orchestra.run.smithi107.stdout:    "mds": {}, 
2020-12-18T17:24:58.319 INFO:teuthology.orchestra.run.smithi107.stdout:    "overall": {
2020-12-18T17:24:58.319 INFO:teuthology.orchestra.run.smithi107.stdout:        "ceph version 15.2.0 (dc6a0b5c3cbf6a5e1d6d4f20b5ad466d76b96247) octopus (rc)": 13
2020-12-18T17:24:58.320 INFO:teuthology.orchestra.run.smithi107.stdout:    }
2020-12-18T17:24:58.320 INFO:teuthology.orchestra.run.smithi107.stdout:}
```
2020-12-18T17:25:02.605 DEBUG:teuthology.orchestra.run.smithi107:> sudo /home/ubuntu/cephtest/cephadm --image docker.io/ceph/ceph:v15.2.0 shell -c /etc/ceph/ceph.conf -k /etc/ceph/ceph.client.admin.keyring --fsid d727547a-4154-11eb-9813-001a4aab830c -e sha1=449059e7a084297c970c09220f6e4730d2d1f991 -- bash -c 'ceph versions | jq -e '"'"'.overall | keys'"'"' | grep $sha1'

File: qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+tasks:
+
+#- install:
+#    branch: nautilus
+
+- cephadm:
+    image: docker.io/ceph/ceph:v15.2.0
+    cephadm_branch: v15.2.0
```
Comment (sebastian-philipp) [inline qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml:8]: want to use latest octopus here?

Comment (yuriw) [reply qa/suites/upgrade/octopus-x/parallel/1-ceph-install/octopus.yaml:8]: Yes!

**Pull request closed without merging**